logstash-core/src/main/java/org/logstash/instrument/witness/PluginsWitness.java

```diff
@@ -20,10 +30,12 @@ public PluginsWitness() {
         this.inputs = new ConcurrentHashMap<>();
         this.outputs = new ConcurrentHashMap<>();
         this.filters = new ConcurrentHashMap<>();
-     }
+        this.codecs = new ConcurrentHashMap<>();
```

Comment:
@jakelandis huh interesting they changed that in Java 8 without me noticing it :D just leave it then yea :)